scripts/revng-check-conventions

```diff
@@ -100,134 +109,244 @@ while [[ $# > 0 ]]; do
     esac
 done
 
```

Comment:
We're invoking `$GIT_LS_FILES` multiple times. We should do it once here.

Reply:
Fixed